ninchatsdk/src/main/java/com/ninchat/sdk/ninchatquestionnaire/ninchatradiobutton/view/NinchatRadioButtonView.kt

```diff
@@ -64,14 +64,14 @@ class NinchatRadioButtonView(
     }
 
     override fun onSelected() {
-        itemView.single_radio_item.setTextColor(ContextCompat.getColor(itemView.context, R.color.ninchat_color_radio_item_selected_text))
-        itemView.single_radio_item.background = ContextCompat.getDrawable(itemView.context, R.drawable.ninchat_radio_select_button)
+       // itemView.single_radio_item.setTextColor(ContextCompat.getColor(itemView.context, R.color.ninchat_color_radio_item_selected_text))
```

Comment:
Why these are commented out?

Reply:
To keep a reference of the previous implementation. I was thinking to come back to this part of the code and see if we can still have the previous codebase without side-effect. But, yes, it's kind of meanless to still have them. I am going to delete it since git history will have them.